src/main/java/org/folio/edge/security/filter/EdgeSecurityFilter.java

```diff
@@ -27,6 +27,9 @@
 @RequiredArgsConstructor
 public class EdgeSecurityFilter extends OncePerRequestFilter {
 
+  public static final String X_TO_CODE = "x-to-code";
+  public static final String X_FROM_CODE = "x-from-code";
+  public static final String AUTHORIZATION_HEADER = "authorization";
```

Comment:
Can you use existing XOkapiHeaders.AUTHORIZATION instead?

Reply:
Yes done and it was case sensitive hence used spring toLowerCase feature along with XOkapiHeaders.AUTHORIZATION